packages/components/toggle-button/src/toggle-button.scss

```diff
@@ -0,0 +1,139 @@
+$fills: outline, ghost;
+$sizes: sm, md, lg;
+$variants: default, primary, success, info, warning, danger;
+
+:host {
+  --_animation: var(--sl-animation-button);
+  --_background: var(--sl-color-button-default-idle-background);
+  --_focus-outline-offset: var(--sl-border-width-focusring-offset);
+  --_focus-outline: var(--sl-color-focusring-default) solid var(--sl-border-width-focusring-default);
+  --_transition-duration: var(--sl-animation-button-duration);
+  --_transition-easing: var(--sl-animation-button-easing);
+
+  align-items: center;
+  background: var(--_background);
+  border: var(--_border-width) solid var(--_border-color);
+  border-radius: var(--_border-radius);
+  color: var(--_color);
+  cursor: pointer;
+  display: inline-flex;
+  font: var(--_font);
+  gap: var(--_label-gap);
+  justify-content: center;
+  outline-color: transparent;
+  padding-block: var(--_padding-block);
+  padding-inline: var(--_padding-inline);
+  transform: var(--_transform);
+  vertical-align: middle;
+}
+
+@mixin color-fill-specific($fill) {
```

Comment:
Since `ghost` is the default fill, shouldn't those styles also be the default here? What if somebody does `fill=""` or something?